src/lib/renderComponent.tsx

```diff
@@ -93,7 +115,7 @@ const renderComponent = <P extends {}>(
         const classes: IComponentPartClasses = getClasses(renderer, mergedStyles, styleParam)
         classes.root = cx(className, classes.root, props.className)
 
-        const accessibility = getAccessibility(props, state)
+        const accessibility = getAccessibility(props, state) as IAccessibilityDefinition
```

Comment:
I would modify `getAccessibility()` to return `IAccessibilityDefinition` and remove the cast here.

Reply:
Agree, done! :)